filament/backend/include/backend/platforms/VulkanPlatform.h

```diff
@@ -27,14 +32,80 @@ namespace filament::backend {
 
 class VulkanPlatform : public Platform {
 public:
+    ~VulkanPlatform() override;
+
     Driver* createDriver(void* const sharedContext,
-	Platform::DriverConfig const& driverConfig) noexcept override;
+            Platform::DriverConfig const& driverConfig) noexcept override;
 
     int getOSVersion() const noexcept override { return 0; }
 
-   ~VulkanPlatform() override;
+
+    /**
+     * Clean up any resources owned by the Platform. For example, if the Vulkan instance handle was
+     * generated by the platform, we need to clean it up in this method.
+     */
+    virtual void cleanup();
```

Comment:
I generally use `terminate()` in filament.

Reply:
renamed.